doc/2/guides/main-concepts/3-querying/index.md

```diff
@@ -8,3 +8,608 @@ order: 300
+
+**Example:** _Search for documents with their ids_ 
+```bash
+kourou document:search ktm-open-data thamel-taxi '{
+  ids: {
+    values: ["aschen", "liia"]
+  }
+}'
+```
+
+::: info
+If you only have an `ids` clause in your search query then you should rather use the [document:mGet](/core/2/api/controllers/document/m-get) action.
+:::
+
+## Boolean Query
+
+It is possible to **combine several clauses in the same query** by using a [Boolean Query](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-bool-query.html).
+
+The following 4 operands are available:
+ - `must`: The clause **must appear** in matching documents and will contribute to the score.
+ - `filter`: The clause **must appear** in matching documents. The score of the query will be ignored.
+ - `should`: The clause **should appear** in the matching document.
+ - `must_not`: The clause **must not appear** in the matching documents. The score of the query will be ignored.
+
+**Example:** _Combining clauses to create an "AND" like search query_
+```bash
+kourou document:search ktm-open-data thamel-taxi '{
+  bool: {
+    filter: [
+      { term: { city: "Tirana" } },
+      { range: { age: { gte: 30 } } }
+    ]
+  }
+}'
+```
+
+**Example:** _Combining clauses to create an "OR" like search query_
+```bash
+kourou document:search ktm-open-data thamel-taxi '{
+  bool: {
+    should: [
+      { term: { city: "Siccieu" } },
+      { term: { city: "Kathmandu" } },
+    ]
+  }
+}'
+```
+
+## Sorting
+
+Elasticsearch offers the possibility to **[sort the results](https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html) by one or more fields**.
+
+**Example:** _Sort by multiple fields_
+```bash
+kourou document:search ktm-open-data thamel-taxi --sort '[
+  { city: "asc" },
+  { age: "desc" }
+]'
+```
+
+A very common sort with Kuzzle is to use [Kuzzle Metadata](/core/2/guides/main-concepts/2-data-storage#kuzzle-metadata) to sort by creation date:
+
+```bash
+kourou document:search ktm-open-data thamel-taxi --sort '[
+  { "_kuzzle_info.createdAt": "asc" }
+]'
+```
+
+Finally, if you want to always ensure the same order for your results you can sort on the `_id` field:
+
+```bash
+kourou document:search ktm-open-data thamel-taxi --sort '[
+  { "_id": "asc" }
+]'
+```
+
+## Pagination
+
+[Several pagination methods](https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html) are available using Elasticsearch and Kuzzle.
+
+They allow to find all documents matching a search query.
+
+::: info
+By default, the [document:search](/core/2/api/controllers/document/search) action returns only 10 documents.  
+The number of returned documents can be changed with the `size` option.  
+There is a limit of 10000 documents that can be retrieved with a single search request, without pagination.
```

Comment:
```suggestion
Elasticsearch does not allow a `search` request to return more than 10000 documents, no matter what pagination parameters are set. Use the `scroll` parameter and the [document:scroll](/core/2/api/controllers/document/scroll) action to paginate more results.
```

Reply:
I updated the comment, tell me what do you think

Reply:
LGTM